db/seeds/08_node_js_course_seeds.rb

```diff
@@ -73,7 +73,7 @@
 
 section_position += 1
 section = create_or_update_section(
-  title: 'Express',
+  title: 'Express & MongoDB',
   title_url: 'Express'.parameterize,
   course_id: course.id,
   position: section_position,
```

Comment:
Can we also update the description since it no longer only indicates that it houses the Express content?

Reply:
I was thinking about that as well. I have not used Express or MongoDB so not sure if the below is good phrasing

How does "Here we finally get to learning the Express framework and integrating MongoDB" sound?

Old: Here we finally get to Express, the most popular back-end JavaScript framework.

Reply:
What about the below: 
> Here we finally get to Express, the most popular back-end JavaScript framework, and MongoDB, a non-relational database frequently paired with Express. 

How do you feel about that?

Reply:
Sounds good to me, I'll update it. Thanks :)